### ISSUE TITLE
(Screen Reader) Sign-in has weird focus

### ISSUE THREAD

**Issue by alexrcabral** [benetech/MathShare#681]
Problem: when using a screen reader, the sign-in button is several layers rather than a single button. 

Solution: The sign-in button should be treated as a link or button rather than a group of one item that has to be entered in/out of. 

((Encountered using voiceover)

**Comment by johnhbenetech** [comment]
@rupeshparab I've also noticed that the '?' and Sign In aren't tab focusable

They should be tab focusable with alt text (help menu, sign in)

They should have the highlighted border and arrow indicator underneath when focused (similar to 'link' and 'google classroom' options)
